security/README.md

```diff
@@ -0,0 +1,53 @@
+Security
+========
+
+A guide for practicing safe web.
+
+Think
+-----
+
+Security is important, and you can't practice these guidelines without
+understanding them. Question each guideline before following it yourself and
+start a discussion whenever you're in doubt.
+
+Using Passwords
+---------------
+
+* Use a unique password for every account you create.
+* Use a tool like [pwgen](https://github.com/jbernard/pwgen) or
+  [1password](https://agilebits.com/onepassword) to generate random passwords.
+* Use a tool like GnuPG to encrypt passwords if you need to share them with
+  somebody.
+
+Accepting Passwords
+-------------------
+
+* Don't store passwords in plain text.
+* Don't hash passwords using a reversible cipher.
+* Don't hash passwords using a broken cipher, such as MD5 or SHA1.
```

Comment:
It seems important enough to call out on its own to me because we're likely to still work with sites available via HTTP. You can have a secure website that uses HTTP but you can't have secure password/session token authentication with HTTP.